packages/jest-util/src/stacktrace.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+// filter for noisy stack trace lines
+const FILTER_NOISE =
+  /(\/|\\)jest(-.*?)?(\/|\\)(vendor|build|node_modules|packages)(\/|\\)/;
+
+const toDiscard = (filename: string, index: number): boolean => {
+  return index !== 0 && FILTER_NOISE.test(filename);
+};
```

Comment:
`toDiscard` isn't a great name for a boolean filter function. How inlining it to avoid giving it a name? It's only a simple expression anyway.